source/fundamentals/promises.txt

```diff
@@ -126,8 +126,18 @@ For additional information, see the MDN documentation on
 :mdn:`await <Web/JavaScript/Reference/Operators/await>`.
 
 
-Callbacks
----------
+Callbacks (Deprecated)
+----------------------
+
+.. important::
+
+    Callbacks are deprecated in the {+driver-short+} version 4.10 and will
+    be removed in a future release. We recommend that you use Promises
+    or Async/Await instead.
+
+    If your application requires the use of callbacks, you can use the
+    {+node-legacy+} driver which wraps the {+driver-short+} and provides
```

Comment:
Issue: `{+node-legacy+}` constant already contains the word "Driver"

Suggestion: 
```suggestion
    {+node-legacy+}, which wraps the {+driver-short+} and provides
```